src/components/Explore/RightSidebar/index.tsx

```diff
@@ -0,0 +1,69 @@
+import { Box, Button, Stack, Typography } from '@mui/material';
+import React, { FC, useState } from 'react';
+import FilterListIcon from '@mui/icons-material/FilterList';
+import { KeyboardDoubleArrowLeftOutlined, KeyboardDoubleArrowRightOutlined, LayersOutlined } from '@mui/icons-material';
+import { theme } from '../../../theme';
+import SearchPanel from './SearchPanel';
+
+const RightSidebar: FC = () => {
+    const [expanded, setExpanded] = useState(false);
+    const [searchPanelOpen, setSearchPanelOpen] = useState(false);
+
+    return (
+        <Stack
+            direction="row"
+            sx={{
+                position: 'absolute',
+                right: 0,
+                height: '100%'
+            }}
+        >
+            <Box sx={{ height: '100%', display: searchPanelOpen ? 'block' : 'none' }}>
+                <SearchPanel onClose={() => setSearchPanelOpen(false)} />
+            </Box>
+            <Stack
+                sx={{
+                    width: expanded ? 118 : 48,
+                    background: theme.palette.explore.main,
+                    backdropFilter: 'blur(4px)'
+                }}
+            >
+                <Button
+                    onClick={() => setSearchPanelOpen(!searchPanelOpen)}
+                    sx={{
+                        justifyContent: 'start',
+                        padding: '12px',
+                        minWidth: 0,
+                        color: theme.palette.explore.secondary,
+                        textTransform: 'none'
+                    }}
+                >
+                    <FilterListIcon />
+                    {expanded ? <Typography ml="12px">Filter</Typography> : null}
+                </Button>
+                <Button
+                    sx={{
+                        justifyContent: 'start',
+                        padding: '12px',
+                        minWidth: 0,
+                        color: theme.palette.explore.secondary,
+                        textTransform: 'none'
+                    }}
+                >
+                    <LayersOutlined />
+                    {expanded ? <Typography ml="12px">Layers</Typography> : null}
```

Comment:
I see the Layer panel is not implemented. Is there any reason why you didn't implement it here? I don't know if you have seen this or not but there is a design that Fangyu had made which looks like this. It can be found [here](https://www.figma.com/file/piCYkwuOYQLGq32CJU2dtC/Challenger-expedition-prototyping?type=design&node-id=9460-73908&mode=design&t=bbM9tG4FMk6WaZ3f-0)
<img width="909" alt="Screenshot 2023-09-13 at 3 50 00 PM" src="https://github.com/Oceans-1876/challenger-portal/assets/32737711/7b778148-3f90-4146-aff6-f9e82f5d3153">